taskcluster/ac_taskgraph/worker_types.py

```diff
@@ -39,7 +42,6 @@ def build_scriptworker_signing_payload(config, task, task_def):
     task_def["tags"]["worker-implementation"] = "scriptworker"
 
     task_def["payload"] = {
-        "maxRunTime": worker["max-run-time"],
```

Comment:
I'm not sure why this is being removed.

Reply:
While sometimes declared in the payload schema, `maxRunTime` isn't used in scriptworker, as far as I know:

* https://github.com/mozilla-releng/signingscript/search?q=maxRunTime&unscoped_q=maxRunTime
* https://github.com/mozilla-releng/beetmoverscript/search?q=maxRunTime&unscoped_q=maxRunTime
* https://github.com/mozilla-releng/scriptworker/search?q=maxRunTime&unscoped_q=maxRunTime

Reply:
:+1: